example/src/App.tsx

```diff
@@ -62,6 +62,7 @@ function getRandomColor() {
 export default function App() {
   const [value, setValue] = React.useState(DEFAULT_TEXT);
   const [markdownStyle, setMarkdownStyle] = React.useState({});
+  const [selection, setSelection] = React.useState({start: 0, end: 0});
```

Comment:
Why are we using state here, but ref in `src/MarkdownTextInput.web.tsx`? Maybe we could standardize it a little?

Reply:
I wanted to simulate how [Expensify NewDot App behaves](https://github.com/Expensify/App/blob/9c49ab65db3f40e719c75516d5655f11b68c0f87/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx#L276) and how users might use it. I'm not sure if we should standardize example app with the MarkdownInput component, since it can be used in various scenarios, for example with selection in the state